node/gum/README.md

```diff
@@ -0,0 +1,25 @@
+# tracing-gum
+
+"gum" to make `tracing::{warn,info,..}` and `mick-jaeger` stick together, to be cross referenced in grafana with zero additional loc in the source code.
+
+## Architecture Decision Record (ADR)
```

Comment:
:+1:

Reply:
Updated the ADR with some more details, which should address the fact this being relevant only to `CandidateHash`.